src/gatling/simulations/uk/gov/hmcts/reform/exui/performance/scenarios/utils/FR_Applicant_Header.scala

```diff
@@ -0,0 +1,286 @@
+package uk.gov.hmcts.reform.exui.performance.scenarios.utils
+
+object FR_Applicant_Header {
+
+  val headers_2 = Map(
+    "Accept" -> "application/json",
+    "Content-Type" -> "application/json",
+    "Sec-Fetch-Dest" -> "empty",
+    "Sec-Fetch-Mode" -> "cors",
+    "Sec-Fetch-Site" -> "same-origin",
+    "X-XSRF-TOKEN" -> "${XSRFToken}",
+    "sec-ch-ua" -> """Chromium";v="86", "\"Not\\A;Brand";v="99", "Google Chrome";v="86""",
+    "sec-ch-ua-mobile" -> "?0",
+    "x-dtpc" -> "3$259808026_422h5vQEGKFHUHMIDPKBQASVVNRMPKURVRUITM-0e16",
+    "x-dtreferer" -> "https://manage-case.perftest.platform.hmcts.net/cases")
+
+  val headers_6 = Map(
+    "Accept" -> "application/vnd.uk.gov.hmcts.ccd-data-store-api.ui-start-case-trigger.v2+json;charset=UTF-8",
+    "Content-Type" -> "application/json",
+    "Sec-Fetch-Dest" -> "empty",
+    "Sec-Fetch-Mode" -> "cors",
+    "Sec-Fetch-Site" -> "same-origin",
+    "X-XSRF-TOKEN" -> "${XSRFToken}",
+    "experimental" -> "true",
+    "sec-ch-ua" -> """Chromium";v="86", "\"Not\\A;Brand";v="99", "Google Chrome";v="86""",
+    "sec-ch-ua-mobile" -> "?0",
+    "x-dtpc" -> "3$259808026_422h8vQEGKFHUHMIDPKBQASVVNRMPKURVRUITM-0e17")
+
+  val headers_8 = Map(
+    "Accept" -> "application/vnd.uk.gov.hmcts.ccd-data-store-api.ui-start-case-trigger.v2+json;charset=UTF-8",
+    "Content-Type" -> "application/json",
+    "Sec-Fetch-Dest" -> "empty",
+    "Sec-Fetch-Mode" -> "cors",
+    "Sec-Fetch-Site" -> "same-origin",
+    "X-XSRF-TOKEN" -> "${XSRFToken}",
+    "experimental" -> "true",
+    "sec-ch-ua" -> """Chromium";v="86", "\"Not\\A;Brand";v="99", "Google Chrome";v="86""",
+    "sec-ch-ua-mobile" -> "?0",
+    "x-dtpc" -> "3$259808026_422h10vQEGKFHUHMIDPKBQASVVNRMPKURVRUITM-0e17")
+
+  val headers_9 = Map(
+    "Accept" -> "application/vnd.uk.gov.hmcts.ccd-data-store-api.ui-user-profile.v2+json;charset=UTF-8",
+    "Content-Type" -> "application/json",
+    "Sec-Fetch-Dest" -> "empty",
+    "Sec-Fetch-Mode" -> "cors",
+    "Sec-Fetch-Site" -> "same-origin",
+    "X-XSRF-TOKEN" -> "${XSRFToken}",
+    "experimental" -> "true",
+    "sec-ch-ua" -> """Chromium";v="86", "\"Not\\A;Brand";v="99", "Google Chrome";v="86""",
+    "sec-ch-ua-mobile" -> "?0",
+    "x-dtpc" -> "3$259808026_422h11vQEGKFHUHMIDPKBQASVVNRMPKURVRUITM-0e17",
+    "x-dtreferer" -> "https://manage-case.perftest.platform.hmcts.net/cases/case-filter")
```

Comment:
parameterise this please and any others you find

Reply:
Done.